examples/bert/npu/user_script.py

```diff
@@ -0,0 +1,19 @@
+from olive.data.component.dataloader import default_dataloader
+from olive.data.registry import Registry
+from olive.platform_sdk.qualcomm.utils.data_loader import FileListCommonDataLoader
+
+
+@Registry.register_dataloader()
+def huggingface_data_to_raw_dataloader(dataset, batch_size=1, io_config=None, **kwargs):
```

Comment:
no need the default batch_size since the caller will pass it. adding a default value will be confusing.

Reply:
Fixed.